scripts/clean_validate.py

```diff
@@ -0,0 +1,95 @@
+# clean_validate.py
+# author: Weilin Han
+# date: 2024-10-20
+
+import click
+import os
+import sys
+import re
+import pandas as pd
+import great_expectations as gx
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from src.clean_data import *
+from src.validate_data import *
+
+@click.command()
+@click.option('--raw-data', type=str, help="Path to directory where raw data resides")
```

Comment:
I think this should be the filename, not the path. What if the filename changes? Then this script would break.

Reply:
try:
```
@click.option('--raw-data-file', type=str, help="Path to raw data file")
```